tests/MongoDB.Analyzer.Tests/Infrastructure/PathUtilities.cs

```diff
@@ -20,17 +20,18 @@ internal static class PathUtilities
 {
     private static readonly string s_testCasesBaseNamespace = "MongoDB.Analyzer.Tests.Common.TestCases";
     private static readonly string s_testCasesBaseFolder = s_testCasesBaseNamespace;
-    private static readonly string s_projectParentFolderPrefix = @"..\..\..\..\";
+    private static readonly string[] s_parentPrefixArray = { "..", "..", "..", "..", " " };
+    private static readonly string s_projectParentFolderPrefix = Path.Combine(s_parentPrefixArray).Trim();
```

Comment:
You could skip having `s_parentPrefixArray` variable. just `Path.Combine(new [] {...})`.
Also why do we need empty prefix `" "`, and `Trim()` ?

Reply:
The reason for this one is as follows: When you run Path.Combine("..", "..", "..", "..") , it will give "../../../.." 

There is no trailing slash at the end. Hence, if we were to do "../../../.." + "A/B/C", we would get "../../../..A/B/C" which isn't what we want. We need "../../../../A/B/C". This is why I put the " " in the parentPrefixArray. 

After putting " " in the parent Array, Path.Combine("..", "..", "..", "..", " ") will give "../../../../ " but we will need to get rid of this extra space at the end when adding this path prefix to other paths. "../../../../ A/B/C" won't resolve to a proper path. "../../../../A/B/C" will resolve to a proper path. 

Hence, as recommended, instead of having a separate variable called s_parentPrefixArray, I just put the array directly inside Path.Combine().

Reply:
I see, in this case we should fix this by using `Path.Combine` in `GetFullPathRelativeToParent`.